bot/exts/moderation/stream.py

```diff
@@ -0,0 +1,139 @@
+import datetime
+
+import discord
+from async_rediscache import RedisCache
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Emojis, Guild, Roles, STAFF_ROLES, VideoPermission
+from bot.converters import Expiry
+from bot.utils.scheduling import Scheduler
+from bot.utils.time import format_infraction_with_duration
+
+
+class Stream(commands.Cog):
+    """Grant and revoke streaming permissions from users."""
+
+    # Stores tasks to remove streaming permission
+    # User id : timestamp relation
+    task_cache = RedisCache()
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        self.scheduler = Scheduler(self.__class__.__name__)
+        self.reload_task = self.bot.loop.create_task(self._reload_tasks_from_redis())
+
+    async def _remove_streaming_permission(self, schedule_user: discord.Member) -> None:
+        """Remove streaming permission from Member."""
+        await self._delete_from_redis(schedule_user.id)
+        await schedule_user.remove_roles(discord.Object(Roles.video), reason="Streaming access revoked")
+
+    async def _add_to_redis_cache(self, user_id: int, timestamp: float) -> None:
+        """Adds 'task' to redis cache."""
+        await self.task_cache.set(user_id, timestamp)
```

Comment:
This function is redundant. All it does is forward arguments to another function without changing anything.